build/params_2k.go

```diff
@@ -16,31 +16,41 @@ import (
 	"github.com/filecoin-project/lotus/chain/actors/policy"
 )
 
-const BootstrappersFile = ""
-const GenesisFile = ""
+const (
```

Comment:
can we avoid these noise cosmetic changes?

Please tell your IDE to not be overeager with formatting.